client/src/components/Dashboard.js

```diff
@@ -59,7 +60,13 @@ export class Dashboard extends Component {
                     <Table.Cell>{dueDate}</Table.Cell>
                     <Table.Cell>{from_language_name}</Table.Cell>
                     <Table.Cell>{to_language_name}</Table.Cell>
-                    <Table.Cell>{status}</Table.Cell>
+                    {status === "Processing" ? (
+                      <Table.Cell>
+                        {status} by {translator_name}
```

Comment:
You could just write "Processing by" here since you know what `status` contains.

Reply:
i think when status is "Completed" we still want to know by whom. Not sure if have right idea about this.

Reply:
I've changed { status !== "Waiting" ? ...} because when status is Completed we still want to know by whom documented was translated.